.github/workflows/tests.yaml

```diff
@@ -59,9 +59,38 @@ jobs:
     - name: Run PHPCS Code Style Checker
       run: dev/sh/style
 
+  docfx:
+    name: "Run DocFX tests"
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - name: Setup Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: '3.8'
+    - run: pip install gcp-docuploader
+    - name: Setup PHP
+      uses: shivammathur/setup-php@verbose
+      with:
+        php-version: "8.0"
+    - name: Install Dependencies
+      uses: nick-invision/retry@v2
+      with:
+        timeout_minutes: 10
+        max_attempts: 3
+        command: composer --no-interaction --no-ansi --no-progress update
+    - name: Install phpDocumentor
+      run: |
+        wget -O /usr/local/bin/phpdoc "https://github.com/phpDocumentor/phpDocumentor/releases/download/v3.3.1/phpDocumentor.phar" \
+          && chmod +x /usr/local/bin/phpdoc
+    - name: Run DocFX Unit Test Suite
+      run: vendor/bin/phpunit dev/tests/Unit/DocFx/
+    - name: Run Docs Generator (Dry Run)
+      run: bash .kokoro/publish-docs.sh
```

Comment:
It seems like dry run doesn't take a bucket and therefore won't upload any docs - am I understanding this correctly?

Reply:
Yes, that was my concern too. Good to hear that's not the case :) Thank you for clarifying!